lib/matplotlib/tests/test_simplification.py

```diff
@@ -54,6 +54,7 @@ def test_noise():
     fig, ax = plt.subplots()
     p1 = ax.plot(x, solid_joinstyle='round', linewidth=2.0)
 
+    fig.canvas.draw()
```

Comment:
added comment

Reply:
You've added the comment to another draw :smile:. There are three of them in the PR.

Each of them is before a `get_path()` would it be feasible/make sense to `_request_autoscale_view()` in this method?

Reply:
They've all been commented now.
What really matters is to have it before get_transform(), but IIRC(?) we can't always update there (because the point is to keep TransformedBboxes computed from get_transform() well, not updated until we perform all updates at the same time in a single shot).